src/modules/seventv/global-emotes.js

```diff
@@ -0,0 +1,63 @@
+import watcher from '../../watcher.js';
+import settings from '../../settings.js';
+import AbstractEmotes from '../emotes/abstract-emotes.js';
+import {createEmote} from './utils.js';
+import {EmoteCategories, EmoteProviders, EmoteTypeFlags, SettingIds} from '../../constants.js';
+import {hasFlag} from '../../utils/flags.js';
+import formatMessage from '../../i18n/index.js';
+
+const category = {
+  id: EmoteCategories.SEVENTV_GLOBAL,
+  provider: EmoteProviders.SEVENTV,
+  displayName: formatMessage({defaultMessage: '7TV Global Emotes'}),
+};
+
+let eventSource;
+
+class SevenTVGlobalEmotes extends AbstractEmotes {
+  constructor() {
+    super();
+
+    watcher.on('channel.updated', () => this.updateGlobalEmotes());
+    settings.on(`changed.${SettingIds.EMOTES}`, () => this.updateGlobalEmotes());
+  }
+
+  get category() {
+    return category;
+  }
+
+  updateGlobalEmotes() {
+    if (eventSource != null) {
```

Comment:
there does not appear to be an eventsource in this file at all. could we delete this and its other references?